examples/README.md

```diff
@@ -2,4 +2,4 @@
 
 This directory contains a set of examples of using various Elastic Cloud resources with Terraform. The examples each have their own README containing more details on what the example does.
 
-To run any example, clone the repository and run `terraform apply` within the example's own directory.
+To run any example clone the repository. Build the provider using `make install` from the main folder. From within the example's directory run `terrafrom init` to initialize Terrafrom, and `terraform apply` to see it work.
```

Comment:
```suggestion
To run any example, clone the repository. Build the provider using `make install` from the main folder. From within the example's directory, run `terraform init` to initialize Terraform, and `terraform apply` to see if it works.
```

Reply:
@alaudazzi I'm not sure how I feel about the phrase `to see if it works` 😄 . To me it sounds like we are giving the impression that our example may or may not work 🤔. The initial intent of the original sentence was to say something like "Run this to see it in action" or to witness it working. 

Do you think maybe there is another way to phrase your suggestion for this sentence?

Reply:
I just saw you changed it for `to see how it works` here https://github.com/elastic/terraform-provider-ec/pull/186/files#diff-150772696d1ea70df8dc2f6fe2c564dd3b72d5b93efc46f9baf9009333d60f00R8

That works for me!